networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -381,7 +383,8 @@ def generate_config(self):
                                                                               status='active')))
         interconnect_hostgroups = self.get_interconnect_hostgroups(nb_switches)
         asn_region = self.get_asn_region(self.region)
-        switchgroups = self.get_switchgroups(asn_region, nb_switches, switch_user, switch_password)
+        switchgroups = sorted(self.get_switchgroups(asn_region, nb_switches, switch_user, switch_password),
+                              key=lambda x: x.name)
```

Comment:
Are the switches inside a switchgroup already sorted?

Reply:
https://github.com/sapcc/networking-ccloud/pull/39/files/efe3f6d9e6d3baf48786e179c0c571c93e428c7b..9189e2fdefd0f26201f31617f70bb7a592c5cb3c#diff-b211a5bc1049300b3d41501d74797736c0411808a9fa4829d026ffba41a6e515R239
Yes they are.